core/src/main/java/org/zenoss/zep/dao/impl/EventSummaryDaoImpl.java

```diff
@@ -26,17 +29,8 @@
 import org.springframework.util.StringUtils;
 import org.zenoss.protobufs.JsonFormat;
 import org.zenoss.protobufs.model.Model.ModelElementType;
-import org.zenoss.protobufs.zep.Zep.Event;
-import org.zenoss.protobufs.zep.Zep.EventActor;
-import org.zenoss.protobufs.zep.Zep.EventAuditLog;
-import org.zenoss.protobufs.zep.Zep.EventDetail;
-import org.zenoss.protobufs.zep.Zep.EventDetailSet;
-import org.zenoss.protobufs.zep.Zep.EventNote;
-import org.zenoss.protobufs.zep.Zep.EventSeverity;
-import org.zenoss.protobufs.zep.Zep.EventStatus;
-import org.zenoss.protobufs.zep.Zep.EventSummary;
-import org.zenoss.protobufs.zep.Zep.EventSummaryOrBuilder;
-import org.zenoss.zep.StatisticsService;
+import org.zenoss.protobufs.zep.Zep.*;
```

Comment:
Try to avoid this. Configure your IDE to make the threshold of imports 999 or something.